assets/js/keyboardShortcuts.js

```diff
@@ -0,0 +1,36 @@
+$(document).keydown(function(e) {
+    var task_id = GetURLParameter('task_id');
+    var project_id = GetURLParameter('project_id');
+
+    Mousetrap.bind("i", function() {
+        var url = window.location.origin + window.location.pathname + '?controller=board&action=assignToMe&task_id=' + task_id + '&project_id=' + project_id;
```

Comment:
Do not hardcode urls in javascript, use data attributes and generate the url with the helper in php